packages/rich-list-core/src/controller/AddressParser/dftx/accountToUtxos.ts

```diff
@@ -0,0 +1,16 @@
+import { fromScript } from '@defichain/jellyfish-address'
+import { AccountToUtxos, CAccountToUtxos, DfTx } from '@defichain/jellyfish-transaction'
+import { ActiveAddress } from '../ActiveAddress'
+import { DfTxAddressParser } from './_abstract'
+
+export class AccountToUtxosParser extends DfTxAddressParser<AccountToUtxos> {
```

Comment:
Checked the rpc behavior, cannot run away from this to extract all involved addresses. DfTx scriptPubKey will not include `addresses` field. This will become a long term effort to maintain rich list accuracy. On the optimistic side, this is not as complicated as calculating the dftx token output.